grpc/server/src/main/java/io/helidon/grpc/server/GrpcServerImpl.java

```diff
@@ -480,8 +479,8 @@ protected SslContextBuilder sslContextBuilder(SslConfiguration sslConfig) {
                                                                      Metadata headers,
                                                                      ServerCallHandler<ReqT, RespT> next) {
 
-            Context context = Context.current().withValue(ContextKeys.HELIDON_CONTEXT, context());
-            return Contexts.interceptCall(context, call, headers, next);
+            io.grpc.Context context = io.grpc.Context.current().withValue(ContextKeys.HELIDON_CONTEXT, context());
```

Comment:
I think this is incorrect. Please make sure that each request runs in its own context instance.
I would expect that within a request your create a new context with the server context as its parent